### PR TITLE
Added user agent to the client builder

### PR DESCRIPTION
**Pull request by scribe**
Got the build version from the localization file, added it to the client builder.

### CODE REVIEW

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ public static Session createConnection(final NewSessionModel newSessionModel, fi
                             new Credentials(newSessionModel.getAccessKey(), newSessionModel.getSecretKey()))
                     .withHttps(newSessionModel.isUseSSL()).withCertificateVerification(false)
                     .withProxy(newSessionModel.getProxyServer())
```
Comment (GraciesPadre) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java]: A couple thoughts...

1) Can't really tell from the GitHub diff, but does this file have a copyright comment?
2) Should the string "Eon Browser" be in a resource bundle or maybe a macro?  Guessing that that string might show up in other places, like title bar text, but I don't know that.
3) Should there be a unit test to make sure we're picking up the build correctly?

Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java]: I second the comment about a resource file lookup.

Comment (DenverM80) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java]: Many many files are missing the copyright comment.  We need to create a task to add it to all files

Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java]: Lets add the copyright in a different pull request if there are a bunch of files without it.

**Comment by scribe** [comment]
I have fixed the lookup to be from the resource bundle

**Pull request merged**